style.css

```diff
@@ -0,0 +1,271 @@
+html {
+	height: 100%;
+}
+body {
+	height: 100%;
+	background: url(img/background.png) repeat;
+	font: normal 1em 'helvetica';
+	color: #4a545d;
+}
+
+.wrapper{
+	position: relative;
+	min-height: 100%;
+	min-width: 978px;
+	height: auto;
+}
+.page-footer{
+	height: 80px;
+}
+
+/* шапка */
+.container-big{
+	max-width: 1024px;
+	min-width: 960px;
+	height: 100%;
+	margin: 0 auto;
+	position: relative;
+}
+.page-header{
+	height: 119px;
+	background: #eff3f7 bottom center no-repeat;
+	position: relative;
+	margin-bottom: 39px;
+}
+.page-header .header-logo-link{
+	position: absolute;
+	top: 39px;
+	left: 0;
+}
+.page-header .header-logo-link{
+	position: absolute;
+	top: 39px;
+	left: 0;
+}
+.page-header .header-logo-text{
+	position: absolute;
+	top: 39px;
+	left: -9999px;
+}
+.page-header .socials{
+	position: absolute;
+	right: 0;
+	top: 36px;
+}
+
+/* соц ссылки */
+.socials{
+	margin: 0;
+	padding: 0;
+	min-width: 260px;
+}
+.socials .socials-item{
+	list-style: none;
+	display: inline-block;
+	height: 47px;
+	width: 47px;
+	margin-left: 19px;
+}
+.socials .socials-item:first-child{
+	margin-left: 0;
+}
+.socials .socials-link{
+	display: block;
+	height: 100%;
+	width: 100%;
+	text-align: -9999px;
+	background: rgb(109, 117, 130);
+}
+.socials .socials-link.gt {
```

Comment:
You may use classes for socials with special `background-image` separately without parent classes.